vault_contract/src/test/setup.rs

```diff
@@ -48,3 +52,66 @@ pub fn get_vc_setup(env: &Env) -> VCVaultContractTest {
         issuance_contract_address,
     }
 }
+
+pub struct DIDContractTest {
+    pub did_init_args: Vec<Val>,
+    pub did_wasm_hash: BytesN<32>,
+    pub salt: BytesN<32>,
+    pub context: Vec<String>,
+}
+
+pub fn get_did_contract_setup(env: &Env, admin: &Address) -> DIDContractTest {
+    let salt = BytesN::from_array(env, &[0; 32]);
+    let did_wasm_hash = env.deployer().upload_contract_wasm(did_contract::WASM);
+
+    DIDContractTest {
+        did_init_args: did_init_args(env, admin),
+        did_wasm_hash,
+        salt,
+        context: did_context(env),
+    }
+}
+
+pub fn did_init_args(e: &Env, admin: &Address) -> Vec<Val> {
+    let verification_methods = vec![
+        e,
+        VerificationMethodEntry {
+            id: String::from_str(e, "keys-1"),
+            type_: VerificationMethodType::Ed25519VerificationKey2020,
+            public_key_multibase: String::from_str(
+                e,
+                "z6MkgpAN9rsVPXJ6DrrvxcsGzKwjdkVdvjNtbQsRiLfsqmuQ",
+            ),
+            controller: String::from_str(e, ""),
+            verification_relationships: vec![e, VerificationRelationship::KeyAgreement],
+        },
```

Comment:
This VerificationMethodEntry is conceptually malformed, `Ed25519VerificationKey2020` type should not be used with `KeyAgreement` relationship.

Get a correct example from the tests of the [DID Contract](https://github.com/kommitters/soroban-did-contract).